components/Review/BizReviewPage.tsx

```diff
@@ -89,8 +90,15 @@ const reviewData = [
   },
 ];
 function BizReviewPage({ navigation }: BizReviewPageProps) {
+  const [modalVisible, setmodalVisible] = React.useState(false);
```

Comment:
The CSS for this page could use a bit of touching up. Right now, the back button is blocked by the notch and the write a review button is partially cut off by the bottom of my screen. The bushiness's profile picture also isn't displaying.

Reply:
ok, i'll look into the first two. not sure why the business profile picture isn't showing up. @finlaylp are you getting that same error when running on my branch?